packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2526,22 +2526,31 @@ export class ContainerRuntime
 	public createDetachedRootDataStore(
 		pkg: Readonly<string[]>,
 		rootDataStoreId: string,
+		groupId?: string,
 	): IFluidDataStoreContextDetached {
 		if (rootDataStoreId.includes("/")) {
 			throw new UsageError(`Id cannot contain slashes: '${rootDataStoreId}'`);
 		}
-		return this.dataStores.createDetachedDataStoreCore(pkg, true, rootDataStoreId);
+		return this.dataStores.createDetachedDataStoreCore(pkg, true, rootDataStoreId, groupId);
 	}
 
-	public createDetachedDataStore(pkg: Readonly<string[]>): IFluidDataStoreContextDetached {
-		return this.dataStores.createDetachedDataStoreCore(pkg, false);
```

Comment:
Could you please add some comments above APIs? Probably on interface (not on concrete class).
I think it should give very short note on what is groupId and point to some readme.md. It's Ok if readme.md is bare, we can add it later, but it would be great to start creating proper documentation structure here.

Reply:
I added the description in the interface on the apis. Readme we can add a bit later like separately.